RadarSDK/RadarAPIClient.m

```diff
@@ -548,11 +547,10 @@ - (void)ipGeocodeWithCompletionHandler:(RadarIPGeocodeAPICompletionHandler)compl
                           return completionHandler(status, nil, nil);
                       }
 
-                      id countryObj = res[@"country"];
-                      RadarRegion *country = [[RadarRegion alloc] initWithObject:countryObj];
+                      RadarAddress *address = [[RadarAddress alloc] initWithObject:res];
 
-                      if (country) {
-                          return completionHandler(RadarStatusSuccess, res, country);
+                      if (address) {
```

Comment:
@kochis Thanks for pulling this together. I think you'll need to pick the `address` dict out of the response like so:

```
id addressObj = res[@"address"];
RadarAddress *address = [[RadarAddress alloc] initWithObject:addressObj];
```

Reply:
I don't think we're sending down a top-level address response

Reply:
I think we should have it for consistency with the other endpoints - https://github.com/radarlabs/server/pull/2157